apps/admin_app/lib/admin_app_web/templates/prototype/index.html.eex

```diff
@@ -0,0 +1,36 @@
+<div class="row">
+  <div class="col-5">
+    <h2>Protoypes</h2>
+  </div>
+  <div class="col-6" align="right">
+    <%= link("Create New", to: prototype_path(@conn, :new), class: "btn btn-primary")%>
+  </div>
+  <div class="col-1"></div>
+</div>
+<div class="row">
+  <div class="col-12">
+<table class="table">
+  <thead>
+    <tr>
+      <th scope="col"></th>
+      <th scope="col">Name</th>
+      <th actions="col"></th>
+    </tr>
+  </thead>
+  <tbody class="ui-sortable">
+    <%= for item <- @prototypes do %>
```

Comment:
Use `render_many` in templates for collections. example `https://elixircasts.io/partial-templates-with-phoenix`